src/lib/openapi/spec/environment-schema.ts

```diff
@@ -25,12 +25,14 @@ export const environmentSchema = {
         },
         protected: {
             type: 'boolean',
+            example: true,
+            description: '`true` if the environment is a protected environment',
```

Comment:
As in the previous comment. What is a protected environment? If this shows up in a lot of places, we should add it to the docs. I couldn't find anything about protected environments in the docs, so we should probably add it anyway.